internal/pkg/admission/policy-server-deployment.go

```diff
@@ -236,6 +236,10 @@ func (r *Reconciler) deployment(configMapVersion string, policyServer *policiesv
 				Name:  "KUBEWARDEN_POLICIES",
 				Value: filepath.Join(policiesConfigContainerPath, policiesFilename),
 			},
+			{
+				Name:  "KUBEWARDEN_SIGSTORE_CACHE_DIR",
```

Comment:
Does policy-server create this directory?

Reply:
NVM just checked: https://github.com/kubewarden/policy-server/blob/7854a1c0328cd1a4514b097ebb6cceb4106f34ae/src/policy_downloader.rs#L218

Reply:
It does [here](https://github.com/kubewarden/policy-server/blob/c5a061f9798aaaae1ca8b865e4efc60489f0115e/src/policy_downloader.rs#L218) (also, tested).